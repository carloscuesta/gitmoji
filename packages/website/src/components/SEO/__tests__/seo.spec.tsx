import renderer from 'react-test-renderer'

import SEO from '../index'
import * as stubs from './stubs'

jest.mock('next/head', () => 'Head')

describe('SEO', () => {
  it('should render the component with pageTitle and pageUrl', () => {
    const wrapper = renderer.create(<SEO {...stubs.props} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should render the component', () => {
    const wrapper = renderer.create(<SEO />)
    expect(wrapper).toMatchSnapshot()
  })
})
