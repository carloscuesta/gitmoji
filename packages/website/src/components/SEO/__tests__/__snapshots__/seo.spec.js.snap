// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render the component 1`] = `null`;

exports[`Button should render the component with pageTitle and pageUrl 1`] = `null`;
